docs/en/observability/synthetics-manage-retention.asciidoc

```diff
@@ -47,4 +47,4 @@ To find Synthetics data streams:
 . Filter the list of data streams for those containing the term `synthetics`.
 .. In the UI there will be three types of browser data streams: `synthetics-browser-*`, `synthetics-browser.network-*`, and `synthetics-browser.screenshot-*`.
 
-Then, you can refer to https://www.elastic.co/guide/en/fleet/current/data-streams.html#data-streams-ilm-tutorial[Tutorial: Customize data retention for integrations] to learn how to apply a custom {ilm-init} policy to the browser data streams. 
+Then, you can refer to https://www.elastic.co/guide/en/fleet/current/data-streams-ilm-tutorial.html[Tutorial: Customize data retention for integrations] to learn how to apply a custom {ilm-init} policy to the browser data streams. 
```

Comment:
I think this might be less fragile: 

```suggestion
Then, you can refer to {fleet-guide}/data-streams-ilm-tutorial.html[Tutorial: Customize data retention for integrations] to learn how to apply a custom {ilm-init} policy to the browser data streams. 
```